mgmt/RecordsConfig.cc

```diff
@@ -1089,7 +1091,7 @@ static const RecordElement RecordsConfig[] =
   {RECT_CONFIG, "proxy.config.hostdb.timed_round_robin", RECD_INT, "0", RECU_DYNAMIC, RR_NULL, RECC_NULL, NULL, RECA_NULL}
   ,
   //       # how often should the hostdb be synced (seconds)
-  {RECT_CONFIG, "proxy.config.cache.hostdb.sync_frequency", RECD_INT, "120", RECU_DYNAMIC, RR_NULL, RECC_NULL, NULL, RECA_NULL}
+  {RECT_CONFIG, "proxy.config.cache.hostdb.sync_frequency", RECD_INT, "120", RECU_RESTART_TS, RR_NULL, RECC_NULL, NULL, RECA_NULL}
```

Comment:
This wasn't dynamic to begin with (even though it said otherwise). I do agree it'd be good to make dynamic-- so I'll open a ticket to followup on that (https://issues.apache.org/jira/browse/TS-4540), but I don't plan on getting that for this PR.